src/test/java/core/basesyntax/service/impl/ReaderImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.Reader;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class ReaderImplTest {
+    private static final String EMPTY_TEST_FILE =
+            "src/test/resources/emptyTestFile.csv";
+    private static final String TEST_FILE =
+            "src/test/resources/testFile.csv";
+    private static final String VALID_TEST_FILE =
```

Comment:
"valid" means "correct". if you are testing incorrect file path, name it "invalid". pay attention to the namings.